10_fibonacci/solution/fibonacci-solution.js

```diff
@@ -1,10 +1,17 @@
 const fibonacci = function(count) {
-	if (count < 0) return "OOPS"
-	const fibPart = [0, 1];
-	for (let index = 1; index < count; index++) {
-		fibPart.push(fibPart[index] + fibPart[index -1]);
-	}
-	return fibPart[count];
+    if (count < 0) return "OOPS";
+    if (count === 0) return 0;
+
+    let first_prev = 1;
+    let second_prev = 0;
+
+    for (let i = 2; i <= count; i++) {
+        let curr = first_prev + second_prev;
+        second_prev = first_prev;
+        first_prev = curr;
+    }
+
+    return first_prev;
```

Comment:
Can we go with `camelCase` please to align with what we teach?

Reply:
Done